lib/src/androidTest/java/co/nimblehq/alpine_lib/ExampleInstrumentedTest.kt

```diff
@@ -0,0 +1,24 @@
+package co.nimblehq.alpine_lib
+
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.ext.junit.runners.AndroidJUnit4
+
+import org.junit.Test
+import org.junit.runner.RunWith
+
+import org.junit.Assert.*
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+@RunWith(AndroidJUnit4::class)
+class ExampleInstrumentedTest {
+    @Test
+    fun useAppContext() {
+        // Context of the app under test.
+        val appContext = InstrumentationRegistry.getInstrumentation().targetContext
+        assertEquals("co.nimblehq.alpine_lib.test", appContext.packageName)
+    }
+}
```

Comment:
Let's remove this file. It was duplicated here:
![Screen Shot 2565-07-18 at 15 31 44](https://user-images.githubusercontent.com/28002315/179473699-893d7ad5-8505-4adf-bc9b-3dee931840e9.png)

Reply:
@Wadeewee nice catch. 👍🏽  Will do.